---
title: "DE-AoP"
subtitle: "Developmental effects of Apnea of Prematurity"
date: 2019-08

author:
  - name: Neural and Neuroencodrine Differenciation & Communication (DC2N)
    url: http://dc2n.labos.univ-rouen.fr/
  - name: Cancer and Brain Genomics (CBG)
    url: https://recherche.univ-rouen.fr/cbg-cancer-and-brain-genomics-umr-1245-679263.kjsp
  - name: Neuroendocrine, Endocrine and Germinal Differentiation Communication (NorDic)
    url: https://sciences-techniques.univ-rouen.fr/nordic-neuroendocrine-endocrine-and-germinal-differentiation-communication-umr-1239-679078.kjsp

image: feature.png
image-alt: "Circular plot illustrating the impact of the multiple genes tested on several key biological pathways"

description: "This project studies the underlying molecular and cellular mechanisms of apnea of prematurity at play during cerebellar development, using intermittent hypoxia in a mouse model."

abstract: |
  This work aims at shedding light on the mechanisms underlying cerebellar hypoxic injury. To this end, a transcriptomic study (by RT-qPCR) of genes involved in oxydative stress, cell differentiation, and migration was performed. We analyzed the expression of these genes in different developmental stages (P4, P8, P12, P21 and adults), and in different cell types, using laser capture microdissection to separate cerebellar layers. This project provides cues to better understand the cellular and molecular aspects of AoP-induced cerebellar injury.

website:
  open-graph:
    description: "Molecular and cellular mechanisms of apnea of prematurity at play during cerebellar development"
  twitter-card:
    description: "Molecular and cellular mechanisms of apnea of prematurity at play during cerebellar development"

categories:
  - R
  - Bioinformatics
  - Biostatistics
  - Software Engineering
  - Immunohistochemistry
  - Transcriptomics
  - RT-qPCR

# Dirty trick to get some links/buttons
about:
  template: solana
  links:
    - text: "Article"
      icon: file-pdf
      file: /content/pubs/CRN23/
      aria-label: "See the article illustrating the project"
    - text: "Code"
      icon: github
      url: https://github.com/ma-riviere/DE-AoP-23
      aria-label: "See the GitHub repository hosting the code of the project and related paper"
    - text: "{{< iconify simple-icons quarto >}} Documentation"
      url: https://ma-riviere.github.io/DE-AoP-23/
      aria-label: "See the website documenting the project's data and code"

toc: false
---

{{< include /content/_hr.qmd >}}

![](graphical-abstract.png){fig-alt="Graphical abstract recapitulating the main findings of this project"}

# Summary

The DE-AoP project included RT-qPCR and immunohistochemistry data, which were analysed in [R](https://www.r-project.org/) [@R]. Data were modeled through the Generalized Linear Mixed Model (GLMM) framework, using the [glmmTMB](https://glmmtmb.github.io/glmmTMB/) package [@glmmTMB]. Random intercepts were added to account for the correlation between pseudo-replicates. Temporally-dependent repeated measures were modeled using auto-regressive (AR1) terms.

The optimal likelihood families were selected based on our theoretical understanding of the variable’s properties, and to minimize Aikake's Information Criterion (AIC). Count data (e.g., cell counts, number of maze entries, …) were modeled using a Generalized Poisson likelihood, measures bound at 0 (e.g., cell density, volumes, weights, …) were modeled using a Gamma likelihood, and proportions (e.g., ratios of areas) with a Beta likelihood.

Model diagnostics were done using the [DHARMa](https://florianhartig.github.io/DHARMa/) [@DHARMa] & [performance](https://easystats.github.io/performance/) [@performance] packages, and estimated marginal means/contrasts were computed with the [emmeans](https://github.com/rvlenth/emmeans) package [@emmeans].

:::{.callout-note appearance="simple"}

### My role in this project

**1)** Handled the data processing, modeling, and analysis for both RT-qPCR and immunohistochemistry data (R/Markdown).

**1)** Developed tools to assist the project's researchers in exploring their data. Among those tools, I coded and hosted a modular [Shiny dashboard](https://mar-studio.shinyapps.io/ShinyQPCR2/) to assist in the data exploration process (R/Shiny).

**3)** Participated in writing the [journal article](/content/pubs/CRN23/) summarizing the results of this project [@rodriguez-dubocApneaPrematurityInduces2023a].

**4)** Open-sourced the resulting code on [GitHub](https://github.com/ma-riviere/DE-AoP-23) and registered it on [Zenodo](https://doi.org/10.5281/zenodo.10198569).

**5)** Made [a website](https://ma-riviere.github.io/DE-AoP-23/) documenting and showcasing the project's data, analyses, and results. The website uses Quarto and Javascript to provide an interactive dashboard to explore the data.

:::

# Quarto website

We made a **Quarto website** to showcase the project's data, analyses, and results with interactive plots and tables:

<a href="https://ma-riviere.github.io/DE-AoP-23/" target="_blank" style="display: block; text-decoration: none;">
    <iframe src="https://ma-riviere.github.io/DE-AoP-23/" width="100%" height="1200px" style="pointer-events: none;"></iframe>
</a>
