<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta name="generator" content="quarto-1.8.17">
<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">
<meta name="author" content="Kavli Institute for Systems Neuroscience">
<meta name="dcterms.date" content="2024-03-01">
<meta name="description" content="Reproducible R analysis of immunohistochemistry data from an Alzheimer’s disease rat model, using mixed-effects models to test AD-control contrasts across brain regions and age-groups.">
<title>AD-McGill – Marc-Aurèle Rivière</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
.display.math{display: block; text-align: center; margin: 0.5rem auto;}
/* CSS for syntax highlighting */
html { -webkit-text-size-adjust: 100%; }
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
  }
pre.numberSource { margin-left: 3em;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
  margin-bottom: 0em;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>

<script src="../../../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../../../site_libs/clipboard/clipboard.min.js"></script>
<script src="../../../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../../../site_libs/quarto-search/fuse.min.js"></script>
<script src="../../../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../../../">
<link href="../../../res/hexabrain.svg" rel="icon" type="image/svg+xml">
<script src="../../../site_libs/cookie-consent/cookie-consent.js"></script>
<link href="../../../site_libs/cookie-consent/cookie-consent.css" rel="stylesheet">
<script src="../../../site_libs/quarto-html/quarto.js" type="module"></script>
<script src="../../../site_libs/quarto-html/tabsets/tabsets.js" type="module"></script>
<script src="../../../site_libs/quarto-html/axe/axe-check.js" type="module"></script>
<script src="../../../site_libs/quarto-html/popper.min.js"></script>
<script src="../../../site_libs/quarto-html/tippy.umd.min.js"></script>
<link href="../../../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../../../site_libs/quarto-html/quarto-syntax-highlighting-e408f6448c0c13359207e36bd9eb5760.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-text-highlighting-styles">
<link href="../../../site_libs/quarto-html/quarto-syntax-highlighting-dark-bc2db561b05f2aaf21fd5844861d6efb.css" rel="stylesheet" class="quarto-color-scheme quarto-color-alternate" id="quarto-text-highlighting-styles">
<link href="../../../site_libs/quarto-html/quarto-syntax-highlighting-e408f6448c0c13359207e36bd9eb5760.css" rel="stylesheet" class="quarto-color-scheme-extra" id="quarto-text-highlighting-styles">
<script src="../../../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../../../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../../../site_libs/bootstrap/bootstrap-e6b6e6864f9235aa4fe375da35a2f717.min.css" rel="stylesheet" append-hash="true" class="quarto-color-scheme" id="quarto-bootstrap" data-mode="light">
<link href="../../../site_libs/bootstrap/bootstrap-dark-65e140c3775b7d11d64f3c410de22ea9.min.css" rel="stylesheet" append-hash="true" class="quarto-color-scheme quarto-color-alternate" id="quarto-bootstrap" data-mode="dark">
<link href="../../../site_libs/bootstrap/bootstrap-e6b6e6864f9235aa4fe375da35a2f717.min.css" rel="stylesheet" append-hash="true" class="quarto-color-scheme-extra" id="quarto-bootstrap" data-mode="light">
<link href="../../../site_libs/quarto-contrib/fontawesome6-0.1.0/all.css" rel="stylesheet">
<link href="../../../site_libs/quarto-contrib/fontawesome6-0.1.0/latex-fontsize.css" rel="stylesheet">
<script src="../../../site_libs/quarto-contrib/glightbox/glightbox.min.js"></script>
<link href="../../../site_libs/quarto-contrib/glightbox/glightbox.min.css" rel="stylesheet">
<link href="../../../site_libs/quarto-contrib/glightbox/lightbox.css" rel="stylesheet"><script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 2,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": true,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script><script async="" src="https://www.googletagmanager.com/gtag/js?id=G-QP3K4HM02B"></script><script type="text/plain" cookie-consent="tracking">

window.dataLayer = window.dataLayer || [];
function gtag(){dataLayer.push(arguments);}
gtag('js', new Date());
gtag('config', 'G-QP3K4HM02B', { 'anonymize_ip': true});
</script><script type="text/javascript" charset="UTF-8">
document.addEventListener('DOMContentLoaded', function () {
cookieconsent.run({
  "notice_banner_type":"simple",
  "consent_type":"implied",
  "palette":"light",
  "language":"en",
  "page_load_consent_levels":["strictly-necessary","functionality","tracking","targeting"],
  "notice_banner_reject_button_hide":false,
  "preferences_center_close_button_hide":false,
  "website_name":""
  ,
"language":"en"
  });
});
</script><link href="../../../site_libs/pagedtable-1.1/css/pagedtable.css" rel="stylesheet">
<script src="../../../site_libs/pagedtable-1.1/js/pagedtable.js"></script><link rel="stylesheet" href="../../../libs/css/style.css">
<meta property="og:title" content="AD-McGill – Marc-Aurèle Rivière">
<meta property="og:description" content="Analyzing BNIP3 expression in the brain of an Alzheimer’s disease rat model">
<meta property="og:image" content="https://ma-riviere.com/content/projects/AD-McGill/graphical-abstract.png">
<meta property="og:site_name" content="Marc-Aurèle Rivière">
<meta property="og:locale" content="en_US">
<meta property="og:image:height" content="2574">
<meta property="og:image:width" content="3382">
<meta property="og:image:alt" content="Graphical abstract recapitulating the main findings of this project">
<meta name="twitter:title" content="AD-McGill – Marc-Aurèle Rivière">
<meta name="twitter:description" content="Analyzing BNIP3 expression in the brain of an Alzheimer’s disease rat model">
<meta name="twitter:image" content="https://ma-riviere.com/content/projects/AD-McGill/graphical-abstract.png">
<meta name="twitter:creator" content="@mariviere1">
<meta name="twitter:card" content="summary">
<meta name="twitter:image-height" content="2574">
<meta name="twitter:image-width" content="3382">
<meta name="twitter:image:alt" content="Graphical abstract recapitulating the main findings of this project">
</head>
<body class="nav-fixed quarto-light"><script id="quarto-html-before-body" type="application/javascript">
    const toggleBodyColorMode = (bsSheetEl) => {
      const mode = bsSheetEl.getAttribute("data-mode");
      const bodyEl = window.document.querySelector("body");
      if (mode === "dark") {
        bodyEl.classList.add("quarto-dark");
        bodyEl.classList.remove("quarto-light");
      } else {
        bodyEl.classList.add("quarto-light");
        bodyEl.classList.remove("quarto-dark");
      }
    }
    const toggleBodyColorPrimary = () => {
      const bsSheetEl = window.document.querySelector("link#quarto-bootstrap:not([rel=disabled-stylesheet])");
      if (bsSheetEl) {
        toggleBodyColorMode(bsSheetEl);
      }
    }
    const setColorSchemeToggle = (alternate) => {
      const toggles = window.document.querySelectorAll('.quarto-color-scheme-toggle');
      for (let i=0; i < toggles.length; i++) {
        const toggle = toggles[i];
        if (toggle) {
          if (alternate) {
            toggle.classList.add("alternate");
          } else {
            toggle.classList.remove("alternate");
          }
        }
      }
    };
    const toggleColorMode = (alternate) => {
      // Switch the stylesheets
      const primaryStylesheets = window.document.querySelectorAll('link.quarto-color-scheme:not(.quarto-color-alternate)');
      const alternateStylesheets = window.document.querySelectorAll('link.quarto-color-scheme.quarto-color-alternate');
      manageTransitions('#quarto-margin-sidebar .nav-link', false);
      if (alternate) {
        // note: dark is layered on light, we don't disable primary!
        enableStylesheet(alternateStylesheets);
        for (const sheetNode of alternateStylesheets) {
          if (sheetNode.id === "quarto-bootstrap") {
            toggleBodyColorMode(sheetNode);
          }
        }
      } else {
        disableStylesheet(alternateStylesheets);
        enableStylesheet(primaryStylesheets)
        toggleBodyColorPrimary();
      }
      manageTransitions('#quarto-margin-sidebar .nav-link', true);
      // Switch the toggles
      setColorSchemeToggle(alternate)
      // Hack to workaround the fact that safari doesn't
      // properly recolor the scrollbar when toggling (#1455)
      if (navigator.userAgent.indexOf('Safari') > 0 && navigator.userAgent.indexOf('Chrome') == -1) {
        manageTransitions("body", false);
        window.scrollTo(0, 1);
        setTimeout(() => {
          window.scrollTo(0, 0);
          manageTransitions("body", true);
        }, 40);
      }
    }
    const disableStylesheet = (stylesheets) => {
      for (let i=0; i < stylesheets.length; i++) {
        const stylesheet = stylesheets[i];
        stylesheet.rel = 'disabled-stylesheet';
      }
    }
    const enableStylesheet = (stylesheets) => {
      for (let i=0; i < stylesheets.length; i++) {
        const stylesheet = stylesheets[i];
        if(stylesheet.rel !== 'stylesheet') { // for Chrome, which will still FOUC without this check
          stylesheet.rel = 'stylesheet';
        }
      }
    }
    const manageTransitions = (selector, allowTransitions) => {
      const els = window.document.querySelectorAll(selector);
      for (let i=0; i < els.length; i++) {
        const el = els[i];
        if (allowTransitions) {
          el.classList.remove('notransition');
        } else {
          el.classList.add('notransition');
        }
      }
    }
    const isFileUrl = () => {
      return window.location.protocol === 'file:';
    }
    const hasAlternateSentinel = () => {
      let styleSentinel = getColorSchemeSentinel();
      if (styleSentinel !== null) {
        return styleSentinel === "alternate";
      } else {
        return false;
      }
    }
    const setStyleSentinel = (alternate) => {
      const value = alternate ? "alternate" : "default";
      if (!isFileUrl()) {
        window.localStorage.setItem("quarto-color-scheme", value);
      } else {
        localAlternateSentinel = value;
      }
    }
    const getColorSchemeSentinel = () => {
      if (!isFileUrl()) {
        const storageValue = window.localStorage.getItem("quarto-color-scheme");
        return storageValue != null ? storageValue : localAlternateSentinel;
      } else {
        return localAlternateSentinel;
      }
    }
    const toggleGiscusIfUsed = (isAlternate, darkModeDefault) => {
      const baseTheme = document.querySelector('#giscus-base-theme')?.value ?? 'light';
      const alternateTheme = document.querySelector('#giscus-alt-theme')?.value ?? 'dark';
      let newTheme = '';
      if(authorPrefersDark) {
        newTheme = isAlternate ? baseTheme : alternateTheme;
      } else {
        newTheme = isAlternate ? alternateTheme : baseTheme;
      }
      const changeGiscusTheme = () => {
        // From: https://github.com/giscus/giscus/issues/336
        const sendMessage = (message) => {
          const iframe = document.querySelector('iframe.giscus-frame');
          if (!iframe) return;
          iframe.contentWindow.postMessage({ giscus: message }, 'https://giscus.app');
        }
        sendMessage({
          setConfig: {
            theme: newTheme
          }
        });
      }
      const isGiscussLoaded = window.document.querySelector('iframe.giscus-frame') !== null;
      if (isGiscussLoaded) {
        changeGiscusTheme();
      }
    };
    const authorPrefersDark = false;
    const darkModeDefault = authorPrefersDark;
      document.querySelector('link#quarto-text-highlighting-styles.quarto-color-scheme-extra').rel = 'disabled-stylesheet';
      document.querySelector('link#quarto-bootstrap.quarto-color-scheme-extra').rel = 'disabled-stylesheet';
    let localAlternateSentinel = darkModeDefault ? 'alternate' : 'default';
    // Dark / light mode switch
    window.quartoToggleColorScheme = () => {
      // Read the current dark / light value
      let toAlternate = !hasAlternateSentinel();
      toggleColorMode(toAlternate);
      setStyleSentinel(toAlternate);
      toggleGiscusIfUsed(toAlternate, darkModeDefault);
      window.dispatchEvent(new Event('resize'));
    };
    // Switch to dark mode if need be
    if (hasAlternateSentinel()) {
      toggleColorMode(true);
    } else {
      toggleColorMode(false);
    }
  </script>

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top quarto-banner"><nav class="navbar navbar-expand-md " data-bs-theme="dark"><div class="navbar-container container-fluid">
      <div class="navbar-brand-container mx-auto">
    <a href="../../../index.html" class="navbar-brand navbar-brand-logo">
    <img src="../../../res/hexabrain.svg" alt="Stylized depiction of a brain inside a hexagon" class="navbar-logo light-content"><img src="../../../res/hexabrain.svg" alt="Stylized depiction of a brain inside a hexagon" class="navbar-logo dark-content"></a>
    <a class="navbar-brand" href="../../../index.html">
    <span class="navbar-title">Marc-Aurèle Rivière</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" role="menu" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
<li class="nav-item">
    <a class="nav-link" href="../../../content/projects/index.html"> <i class="bi bi-journal-code" role="img" aria-label="A curated list of code-related projects I was or am involved in">
</i> 
<span class="menu-text">Projects</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../../../content/pubs/index.html"> <i class="bi bi-file-text" role="img" aria-label="A list of my peer-reviewed scientific publications">
</i> 
<span class="menu-text">Publications</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../../../content/blog/index.html"> <i class="bi bi-braces" role="img" aria-label="A list of blog posts related to R, stats, and data science">
</i> 
<span class="menu-text">Blog</span></a>
  </li>  
</ul>
<ul class="navbar-nav navbar-nav-scroll ms-auto">
<li class="nav-item">
    <a class="nav-link" href="../../../content/about.html"> <i class="bi bi-person-vcard" role="img" aria-label="About Me">
</i> 
<span class="menu-text">About Me</span></a>
  </li>  
  <li class="nav-item compact">
    <a class="nav-link" href="../../../content/blog/index.xml"> <i class="bi bi-rss" role="img" aria-label="My blog's RSS feed">
</i> 
<span class="menu-text"></span></a>
  </li>  
</ul>
</div> <!-- /navcollapse -->
            <div class="quarto-navbar-tools">
    <div class="dropdown">
      <a href="" title="" id="quarto-navigation-tool-dropdown-0" class="quarto-navigation-tool dropdown-toggle px-1" data-bs-toggle="dropdown" aria-expanded="false" role="link" aria-label=""><i class="bi bi-github"></i></a>
      <ul class="dropdown-menu" aria-labelledby="quarto-navigation-tool-dropdown-0">
<li>
            <a class="dropdown-item quarto-navbar-tools-item" href="https://github.com/ma-riviere/ma-riviere.com/">
            Source Code
            </a>
          </li>
          <li>
            <a class="dropdown-item quarto-navbar-tools-item" href="https://github.com/ma-riviere/">
            GitHub Profile
            </a>
          </li>
      </ul>
</div>
  <a href="" class="quarto-color-scheme-toggle quarto-navigation-tool  px-1" onclick="window.quartoToggleColorScheme(); return false;" title="Toggle dark mode"><i class="bi"></i></a>
</div>
      </div> <!-- /container-fluid -->
    </nav></header><!-- content --><header id="title-block-header" class="quarto-title-block default page-columns page-full"><div class="quarto-title-banner page-columns page-full">
    <div class="quarto-title column-body">
      <h1 class="title">AD-McGill</h1>
            <p class="subtitle lead">Bnip3 expression in Alzheimer’s disease rat model</p>
                  <div>
        <div class="description">
          Reproducible R analysis of immunohistochemistry data from an Alzheimer’s disease rat model, using mixed-effects models to test AD-control contrasts across brain regions and age-groups.
        </div>
      </div>
                          <div class="quarto-categories">
                <div class="quarto-category">R</div>
                <div class="quarto-category">Bioinformatics</div>
                <div class="quarto-category">Biostatistics</div>
                <div class="quarto-category">Immunohistochemistry</div>
                <div class="quarto-category">Imaging</div>
              </div>
                  </div>
  </div>
  <div class="quarto-title-meta">
      <div>
      <div class="quarto-title-meta-heading">Consortium</div>
      <div class="quarto-title-meta-contents">
               <p><a href="https://www.ntnu.edu/kavli/">Kavli Institute for Systems Neuroscience</a> </p>
            </div>
    </div>
      <div>
      <div class="quarto-title-meta-heading">Began Around</div>
      <div class="quarto-title-meta-contents">
        <p class="date">March 1, 2024</p>
      </div>
    </div>
    </div>
  <div>
    <div class="abstract">
      <div class="block-title">Abstract</div>
      <p>Alzheimer’s disease (AD) is associated with mitochondrial dysfunction and hypoxia-like signaling that can induce BNIP3, a key mediator of mitophagy and cell death. This study quantified BNIP3 immunoreactivity in neuronal soma and axonal projections across multiple brain regions (entorhinal cortex, CA3, dentate gyrus) in an established AD rat model compared to matched controls. We analyzed brightfield immunohistochemistry data from 24 rats (12 AD model, 12 controls) across three age groups (3, 12, 18 months), measuring optical density with background correction.</p>
      <p>The data was analyzed using generalized linear mixed-effects models with Gamma likelihoods and random effects captured animal-level and section-level variability, while fixed effects tested genotype, age, and brain region effects, and their interactions. Model fit was rigorously assessed using diagnostic plots, statistical tests, and posterior predictive checks.</p>
      <p>The repository provides open data, code, and key figures used in the associated article. All figures and tables were generated reproducibly from R Markdown in a version-controlled environment.</p>
    </div>
  </div>
  </header><div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active"><h2 id="toc-title">On this page</h2>
   
  <ul>
<li><a href="#summary" id="toc-summary" class="nav-link active" data-scroll-target="#summary">Summary</a></li>
  <li><a href="#data" id="toc-data" class="nav-link" data-scroll-target="#data">Data</a></li>
  <li><a href="#models" id="toc-models" class="nav-link" data-scroll-target="#models">Models</a></li>
  <li><a href="#hypotheses" id="toc-hypotheses" class="nav-link" data-scroll-target="#hypotheses">Hypotheses</a></li>
  </ul></nav>
    </div>
<!-- main -->
<main class="content quarto-banner-title-block page-columns page-full" id="quarto-document-content"><script>
  let processScroll = () => {
    let docElem = document.documentElement, 
      docBody = document.body,
      scrollTop = docElem['scrollTop'] || docBody['scrollTop'],
      scrollBottom = (docElem['scrollHeight'] || docBody['scrollHeight']) - window.innerHeight,
      scrollPercent = scrollTop / scrollBottom * 100 + '%';

    document.getElementById("progress-bar").style.setProperty("--scrollAmount", scrollPercent); 
  }

  document.addEventListener('scroll', processScroll);

  let navbarel = document.querySelector("#quarto-header");

  let pbc = document.createElement("div");
  pbc.id = "progress-bar";

  navbarel.insertAdjacentElement('beforeend', pbc);
</script><style>
  #progress-bar {
    --scrollAmount: 0%;
    width: var(--scrollAmount);
    height: 5px;
    position: fixed;
    bottom: 0;
  }
</style>
<div class="quarto-about-solana column-body">
  <div class="about-entity">
    <div class="entity-contents">
       <div class="about-links">
  <a href="https://agalic-rd.com/content/pubs/articles/BM25/" class="about-link" rel="" target="">
    <i class="bi bi-file-pdf"></i>
     <span class="about-link-text">Article</span>
  </a>
  <a href="https://github.com/agalic-rd/Bnip3_McGill_AD/" class="about-link" rel="" target="">
    <i class="bi bi-github"></i>
     <span class="about-link-text">Code</span>
  </a>
</div>
      <div class="about-contents"><div id="about-links">
</div></div>
    </div>
    <img src="graphical-abstract.png" alt="Graphical abstract recapitulating the main findings of this project" class="about-image
  rounded " style="width: 0%;">
  </div>
</div>

<hr style="margin-bottom: 0px; margin-top: -12px;" width="1000px">
<section id="summary" class="level1"><h1>Summary</h1>
<p><a href="graphical-abstract.png" class="lightbox" data-gallery="quarto-lightbox-gallery-1"><img src="graphical-abstract.png" class="img-fluid" alt="Graphical abstract recapitulating the main findings of this project"></a></p>
<div class="callout callout-style-simple callout-note callout-titled">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
<span class="screen-reader-only">Note</span>My role in this project
</div>
</div>
<div class="callout-body-container callout-body">
<p><strong>1)</strong> Managed the code for data processing, modeling, and hypothesis testing.</p>
<p><strong>2)</strong> Helped write the methodology/statistics part of the <a href="https://agalic-rd.com/content/pubs/articles/BM25/">paper</a> <span class="citation" data-cites="rodriguez-dubocBnip3ExpressionBrain2025">(<a href="#ref-rodriguez-dubocBnip3ExpressionBrain2025" role="doc-biblioref">Rodriguez-Duboc et al., 2025</a>)</span>.</p>
<p><strong>3)</strong> Open-sourced the resulting code on <a href="https://github.com/agalic-rd/Bnip3_McGill_AD">GitHub</a> and registered it on <a href="https://doi.org/10.5281/zenodo.15199696">Zenodo</a>.</p>
</div>
</div>
</section><section id="data" class="level1"><h1>Data</h1>
<p>The project analyzed brightfield immunohistochemistry (IHC) data quantified as optical density (OD) measurements from brain tissue sections. Data was collected from 24 rats (12 AD model, 12 controls) across three age groups (3, 12, 18 months) to measure BNIP3 protein expression in neuronal soma and axonal projections. Each animal contributed 7-8 tissue sections covering specific brain regions (entorhinal cortex, CA3, dentate gyrus), with background correction applied by subtracting deep layer measurements from raw OD values. The dataset included hierarchical structure with measurements nested within sections, sections within animals, creating pseudo-replication that required mixed-effects modeling approaches.</p>
<p>Data preprocessing involved standardizing variable names and metadata using a data dictionary to ensure consistent analysis workflows. The final dataset captured both cellular (soma) and projection field (molecular layer) measurements across genotype, age, and brain region factors.</p>
</section><section id="models" class="level1"><h1>Models</h1>
<p>Optical Density (OD) data was modeled using a Gamma distribution to account for the data’s strict positivity and positive skewness, with a log link function.</p>
<p>Generalized Linear Mixed Models (GLMMs) were used to model the data, using random effects to account for pseudo-replication and the hierarchical nature of the data. For all models, categorical predictors were dummy-coded with a reference level: genotype (Wild-Type) and age (3 months). Models were fitted using the <a href="https://glmmtmb.github.io/glmmTMB/">glmmTMB</a> package <span class="citation" data-cites="glmmTMB">(<a href="#ref-glmmTMB" role="doc-biblioref">Brooks et al., 2017</a>)</span>.</p>
<p>Model fit was assessed using a combination of plots (normality, homoscedasticity, normality of residuals, collinearity), statistical tests (Levenne, KS, dispersion), and posterior predictive checks on simple statistics (min, max, mean, sd) using simulated residuals. Model fit was assessed using the <a href="https://florianhartig.github.io/DHARMa/">DHARMa</a> <span class="citation" data-cites="DHARMa">(<a href="#ref-DHARMa" role="doc-biblioref">Hartig, 2022</a>)</span> and <a href="https://easystats.github.io/performance/">performance</a> <span class="citation" data-cites="performance">(<a href="#ref-performance" role="doc-biblioref">Lüdecke et al., 2021</a>)</span> packages.</p>
<p>Two models were fitted on the OD data, one for the <strong>soma</strong> and one for the <strong>molecular layer</strong> (terminal fields):</p>
<div class="tabset-margin-container"></div><div class="panel-tabset">
<ul class="nav nav-tabs" role="tablist">
<li class="nav-item" role="presentation"><a class="nav-link active" id="tabset-3-1-tab" data-bs-toggle="tab" data-bs-target="#tabset-3-1" role="tab" aria-controls="tabset-3-1" aria-selected="true">‘Soma’ model</a></li>
<li class="nav-item" role="presentation"><a class="nav-link" id="tabset-3-2-tab" data-bs-toggle="tab" data-bs-target="#tabset-3-2" role="tab" aria-controls="tabset-3-2" aria-selected="false">‘Molecular Layer’ model</a></li>
</ul>
<div class="tab-content">
<div id="tabset-3-1" class="tab-pane active" role="tabpanel" aria-labelledby="tabset-3-1-tab">
<p>We modeled the OD of individual neuronal somata (<img style="vertical-align:middle" src="https://latex.codecogs.com/svg.latex?%5Ctextstyle%20Y_%7Bijk%7D" alt="Y_{ijk}" title="Y_{ijk}" class="math inline">) using genotype, age, and brain area (LEC vs.&nbsp;MEC) as fixed effects, along with all their interactions. Random intercepts were included for each rat (j) and for each tissue section (k) nested within each rat.</p>
<section id="model-equation" class="level3"><h3 data-anchor-id="model-equation">Model equation</h3>
<p><img style="vertical-align:middle" src="https://latex.codecogs.com/svg.latex?%5Cdisplaystyle%20%5Cbegin%7Baligned%7D%0AY_%7Bijk%7D%20%26%5Csim%20%5Cmathrm%7BGamma%7D%5Cbigl%28%5Cmu_%7Bijk%7D%2C%5Cphi%5Cbigr%29%2C%5Cquad%0A%20%20%5Cmathrm%7BVar%7D%28Y_%7Bijk%7D%29%20%3D%20%5Cphi%5C%2C%5Cmu_%7Bijk%7D%5E2%2C%5C%5C%0A%5Clog%28%5Cmu_%7Bijk%7D%29%0A%20%20%20%20%26%3D%20%5Cbeta_0%5C%5C%0A%20%20%20%20%26%5Cquad%20%2B%20%5Cbeta_%7B%5Cmathrm%7BGenotype%7D%7D%5Cbigl%28%5Ctext%7Bgenotype%7D_%7Bijk%7D%5Cbigr%29%5C%5C%0A%20%20%20%20%26%5Cquad%20%2B%20%5Cbeta_%7B%5Cmathrm%7BAge%7D%7D%5Cbigl%28%5Ctext%7Bage%7D_%7Bijk%7D%5Cbigr%29%5C%5C%0A%20%20%20%20%26%5Cquad%20%2B%20%5Cbeta_%7B%5Cmathrm%7BArea%7D%7D%5Cbigl%28%5Ctext%7Barea%7D_%7Bijk%7D%5Cbigr%29%5C%5C%0A%20%20%20%20%26%5Cquad%20%2B%20%5Cbeta_%7B%5Cmathrm%7BGenotype%7D%5Ctimes%5Cmathrm%7BAge%7D%7D%5Cbigl%28%5Ctext%7Bgenotype%7D_%7Bijk%7D%2C%5Ctext%7Bage%7D_%7Bijk%7D%5Cbigr%29%5C%5C%0A%20%20%20%20%26%5Cquad%20%2B%20%5Cbeta_%7B%5Cmathrm%7BGenotype%7D%5Ctimes%5Cmathrm%7BArea%7D%7D%5Cbigl%28%5Ctext%7Bgenotype%7D_%7Bijk%7D%2C%5Ctext%7Barea%7D_%7Bijk%7D%5Cbigr%29%5C%5C%0A%20%20%20%20%26%5Cquad%20%2B%20%5Cbeta_%7B%5Cmathrm%7BAge%7D%5Ctimes%5Cmathrm%7BArea%7D%7D%5Cbigl%28%5Ctext%7Bage%7D_%7Bijk%7D%2C%5Ctext%7Barea%7D_%7Bijk%7D%5Cbigr%29%5C%5C%0A%20%20%20%20%26%5Cquad%20%2B%20%5Cbeta_%7B%5Cmathrm%7BGenotype%7D%5Ctimes%5Cmathrm%7BAge%7D%5Ctimes%5Cmathrm%7BArea%7D%7D%5Cbigl%28%5Ctext%7Bgenotype%7D_%7Bijk%7D%2C%5Ctext%7Bage%7D_%7Bijk%7D%2C%5Ctext%7Barea%7D_%7Bijk%7D%5Cbigr%29%5C%5C%0A%20%20%20%20%26%5Cquad%20%2B%20b_%7B%5Cmathrm%7Brat%7D%5Bj%5D%7D%20%2B%20b_%7B%5Cmathrm%7Brat%3Asection%7D%5Bj%2Ck%5D%7D%2C%5C%5C%0Ab_%7B%5Cmathrm%7Brat%7D%5Bj%5D%7D%20%26%5Csim%20%5Cmathcal%7BN%7D%5Cbigl%280%2C%5Csigma%5E2_%7B%5Cmathrm%7Brat%7D%7D%5Cbigr%29%2C%5Cquad%0Ab_%7B%5Cmathrm%7Brat%3Asection%7D%5Bj%2Ck%5D%7D%20%5Csim%20%5Cmathcal%7BN%7D%5Cbigl%280%2C%5Csigma%5E2_%7B%5Cmathrm%7Brat%3Asection%7D%7D%5Cbigr%29.%0A%5Cend%7Baligned%7D" alt="\begin{aligned}
Y_{ijk} &amp;\sim \mathrm{Gamma}\bigl(\mu_{ijk},\phi\bigr),\quad
  \mathrm{Var}(Y_{ijk}) = \phi\,\mu_{ijk}^2,\\
\log(\mu_{ijk})
    &amp;= \beta_0\\
    &amp;\quad + \beta_{\mathrm{Genotype}}\bigl(\text{genotype}_{ijk}\bigr)\\
    &amp;\quad + \beta_{\mathrm{Age}}\bigl(\text{age}_{ijk}\bigr)\\
    &amp;\quad + \beta_{\mathrm{Area}}\bigl(\text{area}_{ijk}\bigr)\\
    &amp;\quad + \beta_{\mathrm{Genotype}\times\mathrm{Age}}\bigl(\text{genotype}_{ijk},\text{age}_{ijk}\bigr)\\
    &amp;\quad + \beta_{\mathrm{Genotype}\times\mathrm{Area}}\bigl(\text{genotype}_{ijk},\text{area}_{ijk}\bigr)\\
    &amp;\quad + \beta_{\mathrm{Age}\times\mathrm{Area}}\bigl(\text{age}_{ijk},\text{area}_{ijk}\bigr)\\
    &amp;\quad + \beta_{\mathrm{Genotype}\times\mathrm{Age}\times\mathrm{Area}}\bigl(\text{genotype}_{ijk},\text{age}_{ijk},\text{area}_{ijk}\bigr)\\
    &amp;\quad + b_{\mathrm{rat}[j]} + b_{\mathrm{rat:section}[j,k]},\\
b_{\mathrm{rat}[j]} &amp;\sim \mathcal{N}\bigl(0,\sigma^2_{\mathrm{rat}}\bigr),\quad
b_{\mathrm{rat:section}[j,k]} \sim \mathcal{N}\bigl(0,\sigma^2_{\mathrm{rat:section}}\bigr).
\end{aligned}" title="\begin{aligned}
Y_{ijk} &amp;\sim \mathrm{Gamma}\bigl(\mu_{ijk},\phi\bigr),\quad
  \mathrm{Var}(Y_{ijk}) = \phi\,\mu_{ijk}^2,\\
\log(\mu_{ijk})
    &amp;= \beta_0\\
    &amp;\quad + \beta_{\mathrm{Genotype}}\bigl(\text{genotype}_{ijk}\bigr)\\
    &amp;\quad + \beta_{\mathrm{Age}}\bigl(\text{age}_{ijk}\bigr)\\
    &amp;\quad + \beta_{\mathrm{Area}}\bigl(\text{area}_{ijk}\bigr)\\
    &amp;\quad + \beta_{\mathrm{Genotype}\times\mathrm{Age}}\bigl(\text{genotype}_{ijk},\text{age}_{ijk}\bigr)\\
    &amp;\quad + \beta_{\mathrm{Genotype}\times\mathrm{Area}}\bigl(\text{genotype}_{ijk},\text{area}_{ijk}\bigr)\\
    &amp;\quad + \beta_{\mathrm{Age}\times\mathrm{Area}}\bigl(\text{age}_{ijk},\text{area}_{ijk}\bigr)\\
    &amp;\quad + \beta_{\mathrm{Genotype}\times\mathrm{Age}\times\mathrm{Area}}\bigl(\text{genotype}_{ijk},\text{age}_{ijk},\text{area}_{ijk}\bigr)\\
    &amp;\quad + b_{\mathrm{rat}[j]} + b_{\mathrm{rat:section}[j,k]},\\
b_{\mathrm{rat}[j]} &amp;\sim \mathcal{N}\bigl(0,\sigma^2_{\mathrm{rat}}\bigr),\quad
b_{\mathrm{rat:section}[j,k]} \sim \mathcal{N}\bigl(0,\sigma^2_{\mathrm{rat:section}}\bigr).
\end{aligned}" class="math display"></p>
<div class="callout callout-style-simple callout-note callout-titled">
<div class="callout-header d-flex align-content-center collapsed" data-bs-toggle="collapse" data-bs-target=".callout-2-contents" aria-controls="callout-2" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
<span class="screen-reader-only">Note</span>Equivalent R model
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-2" class="callout-2-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<div class="cell" data-layout-align="center">
<div class="code-copy-outer-scaffold"><div class="sourceCode" id="cb1"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="fu">glmmTMB</span><span class="fu">::</span><span class="fu">glmmTMB</span><span class="op">(</span></span>
<span>  <span class="va">od_corrected</span> <span class="op">~</span> <span class="va">genotype</span> <span class="op">*</span> <span class="va">age</span> <span class="op">*</span> <span class="va">area</span> <span class="op">+</span> <span class="op">(</span><span class="fl">1</span> <span class="op">|</span> <span class="va">rat</span> <span class="op">/</span> <span class="va">section</span><span class="op">)</span>,</span>
<span>  family <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/stats/family.html">Gamma</a></span><span class="op">(</span><span class="st">"log"</span><span class="op">)</span>,</span>
<span>  data <span class="op">=</span> <span class="va">od_data</span><span class="op">$</span><span class="va">soma</span>,</span>
<span>  REML <span class="op">=</span> <span class="cn">TRUE</span></span>
<span><span class="op">)</span></span></code></pre></div><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></div>
</div>
</div>
</div>
</div>
</section><section id="model-fit" class="level3"><h3 data-anchor-id="model-fit">Model fit</h3>
<div class="tabset-margin-container"></div><div class="panel-tabset">
<ul class="nav nav-tabs" role="tablist">
<li class="nav-item" role="presentation"><a class="nav-link active" id="tabset-1-1-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-1" role="tab" aria-controls="tabset-1-1" aria-selected="true">Plots</a></li>
<li class="nav-item" role="presentation"><a class="nav-link" id="tabset-1-2-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-2" role="tab" aria-controls="tabset-1-2" aria-selected="false">Tests</a></li>
<li class="nav-item" role="presentation"><a class="nav-link" id="tabset-1-3-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-3" role="tab" aria-controls="tabset-1-3" aria-selected="false">Posterior Predictive Checks</a></li>
</ul>
<div class="tab-content">
<div id="tabset-1-1" class="tab-pane active" role="tabpanel" aria-labelledby="tabset-1-1-tab">
<div class="quarto-figure quarto-figure-center">
<figure class="figure"><p><a href="mod_soma_diag.png" class="lightbox" data-gallery="quarto-lightbox-gallery-2"><img src="mod_soma_diag.png" class="img-fluid quarto-figure quarto-figure-center figure-img" alt="Performance plots for the 'Soma' model"></a></p>
</figure>
</div>
</div>
<div id="tabset-1-2" class="tab-pane" role="tabpanel" aria-labelledby="tabset-1-2-tab">
<div class="quarto-figure quarto-figure-center">
<figure class="figure"><p><a href="mod_soma_dharma.png" class="lightbox" data-gallery="quarto-lightbox-gallery-3"><img src="mod_soma_dharma.png" class="img-fluid quarto-figure quarto-figure-center figure-img" alt="DHARMa plots for the 'Soma' model"></a></p>
</figure>
</div>
</div>
<div id="tabset-1-3" class="tab-pane" role="tabpanel" aria-labelledby="tabset-1-3-tab">
<div class="quarto-figure quarto-figure-center">
<figure class="figure"><p><a href="mod_soma_age_ppc_stats.png" class="lightbox" data-gallery="quarto-lightbox-gallery-4"><img src="mod_soma_age_ppc_stats.png" class="img-fluid quarto-figure quarto-figure-center figure-img" alt="Posterior Predictive Checks on simple stats for the 'Soma' model"></a></p>
</figure>
</div>
</div>
</div>
</div>
</section>
</div>
<div id="tabset-3-2" class="tab-pane" role="tabpanel" aria-labelledby="tabset-3-2-tab">
<p>To model the OD of the terminal fields (<img style="vertical-align:middle" src="https://latex.codecogs.com/svg.latex?%5Ctextstyle%20Y_%7Bij%7D" alt="Y_{ij}" title="Y_{ij}" class="math inline">), the fixed effects included genotype, age, and their interaction. A random intercept for each rat (j) was included to account for inter-animal variability.</p>
<section id="model-equation-1" class="level3"><h3 data-anchor-id="model-equation-1">Model equation</h3>
<p><img style="vertical-align:middle" src="https://latex.codecogs.com/svg.latex?%5Cdisplaystyle%20%5Cbegin%7Baligned%7D%0AY_%7Bij%7D%20%26%5Csim%20%5Cmathrm%7BGamma%7D%5Cbigl%28%5Cmu_%7Bij%7D%2C%5Cphi%5Cbigr%29%2C%5Cquad%0A%20%20%5Cmathrm%7BVar%7D%28Y_%7Bij%7D%29%20%3D%20%5Cphi%5C%2C%5Cmu_%7Bij%7D%5E2%2C%5C%5C%0A%5Clog%28%5Cmu_%7Bij%7D%29%0A%20%20%20%20%26%3D%20%5Cbeta_0%5C%5C%0A%20%20%20%20%26%5Cquad%20%2B%20%5Cbeta_%7B%5Cmathrm%7BGenotype%7D%7D%5Cbigl%28%5Ctext%7Bgenotype%7D_%7Bij%7D%5Cbigr%29%5C%5C%0A%20%20%20%20%26%5Cquad%20%2B%20%5Cbeta_%7B%5Cmathrm%7BAge%7D%7D%5Cbigl%28%5Ctext%7Bage%7D_%7Bij%7D%5Cbigr%29%5C%5C%0A%20%20%20%20%26%5Cquad%20%2B%20%5Cbeta_%7B%5Cmathrm%7BGenotype%7D%5Ctimes%5Cmathrm%7BAge%7D%7D%5Cbigl%28%5Ctext%7Bgenotype%7D_%7Bij%7D%2C%5Ctext%7Bage%7D_%7Bij%7D%5Cbigr%29%5C%5C%0A%20%20%20%20%26%5Cquad%20%2B%20b_%7B%5Cmathrm%7Brat%7D%5Bj%5D%7D%2C%5C%5C%0Ab_%7B%5Cmathrm%7Brat%7D%5Bj%5D%7D%20%26%5Csim%20%5Cmathcal%7BN%7D%5Cbigl%280%2C%5Csigma%5E2_%7B%5Cmathrm%7Brat%7D%7D%5Cbigr%29.%0A%5Cend%7Baligned%7D" alt="\begin{aligned}
Y_{ij} &amp;\sim \mathrm{Gamma}\bigl(\mu_{ij},\phi\bigr),\quad
  \mathrm{Var}(Y_{ij}) = \phi\,\mu_{ij}^2,\\
\log(\mu_{ij})
    &amp;= \beta_0\\
    &amp;\quad + \beta_{\mathrm{Genotype}}\bigl(\text{genotype}_{ij}\bigr)\\
    &amp;\quad + \beta_{\mathrm{Age}}\bigl(\text{age}_{ij}\bigr)\\
    &amp;\quad + \beta_{\mathrm{Genotype}\times\mathrm{Age}}\bigl(\text{genotype}_{ij},\text{age}_{ij}\bigr)\\
    &amp;\quad + b_{\mathrm{rat}[j]},\\
b_{\mathrm{rat}[j]} &amp;\sim \mathcal{N}\bigl(0,\sigma^2_{\mathrm{rat}}\bigr).
\end{aligned}" title="\begin{aligned}
Y_{ij} &amp;\sim \mathrm{Gamma}\bigl(\mu_{ij},\phi\bigr),\quad
  \mathrm{Var}(Y_{ij}) = \phi\,\mu_{ij}^2,\\
\log(\mu_{ij})
    &amp;= \beta_0\\
    &amp;\quad + \beta_{\mathrm{Genotype}}\bigl(\text{genotype}_{ij}\bigr)\\
    &amp;\quad + \beta_{\mathrm{Age}}\bigl(\text{age}_{ij}\bigr)\\
    &amp;\quad + \beta_{\mathrm{Genotype}\times\mathrm{Age}}\bigl(\text{genotype}_{ij},\text{age}_{ij}\bigr)\\
    &amp;\quad + b_{\mathrm{rat}[j]},\\
b_{\mathrm{rat}[j]} &amp;\sim \mathcal{N}\bigl(0,\sigma^2_{\mathrm{rat}}\bigr).
\end{aligned}" class="math display"></p>
<div class="callout callout-style-simple callout-note callout-titled">
<div class="callout-header d-flex align-content-center collapsed" data-bs-toggle="collapse" data-bs-target=".callout-3-contents" aria-controls="callout-3" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
<span class="screen-reader-only">Note</span>Equivalent R model
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-3" class="callout-3-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<div class="cell" data-layout-align="center">
<div class="code-copy-outer-scaffold"><div class="sourceCode" id="cb2"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="fu">glmmTMB</span><span class="op">(</span></span>
<span>    <span class="va">od_corrected</span> <span class="op">~</span> <span class="va">genotype</span> <span class="op">*</span> <span class="va">age</span> <span class="op">+</span> <span class="op">(</span><span class="fl">1</span> <span class="op">|</span> <span class="va">rat</span><span class="op">)</span>,</span>
<span>    family <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/stats/family.html">Gamma</a></span><span class="op">(</span><span class="st">"log"</span><span class="op">)</span>,</span>
<span>    data <span class="op">=</span> <span class="va">od_data</span><span class="op">$</span><span class="va">ml</span>,</span>
<span>    REML <span class="op">=</span> <span class="cn">TRUE</span></span>
<span><span class="op">)</span></span></code></pre></div><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></div>
</div>
</div>
</div>
</div>
</section><section id="model-fit-1" class="level3"><h3 data-anchor-id="model-fit-1">Model fit</h3>
<div class="tabset-margin-container"></div><div class="panel-tabset">
<ul class="nav nav-tabs" role="tablist">
<li class="nav-item" role="presentation"><a class="nav-link active" id="tabset-2-1-tab" data-bs-toggle="tab" data-bs-target="#tabset-2-1" role="tab" aria-controls="tabset-2-1" aria-selected="true">Plots</a></li>
<li class="nav-item" role="presentation"><a class="nav-link" id="tabset-2-2-tab" data-bs-toggle="tab" data-bs-target="#tabset-2-2" role="tab" aria-controls="tabset-2-2" aria-selected="false">Tests</a></li>
<li class="nav-item" role="presentation"><a class="nav-link" id="tabset-2-3-tab" data-bs-toggle="tab" data-bs-target="#tabset-2-3" role="tab" aria-controls="tabset-2-3" aria-selected="false">Posterior Predictive Checks</a></li>
</ul>
<div class="tab-content">
<div id="tabset-2-1" class="tab-pane active" role="tabpanel" aria-labelledby="tabset-2-1-tab">
<div class="quarto-figure quarto-figure-center">
<figure class="figure"><p><a href="mod_ml_diag.png" class="lightbox" data-gallery="quarto-lightbox-gallery-5"><img src="mod_ml_diag.png" class="img-fluid quarto-figure quarto-figure-center figure-img" alt="Performance plots for the 'Molecular Layer' model"></a></p>
</figure>
</div>
</div>
<div id="tabset-2-2" class="tab-pane" role="tabpanel" aria-labelledby="tabset-2-2-tab">
<div class="quarto-figure quarto-figure-center">
<figure class="figure"><p><a href="mod_ml_dharma.png" class="lightbox" data-gallery="quarto-lightbox-gallery-6"><img src="mod_ml_dharma.png" class="img-fluid quarto-figure quarto-figure-center figure-img" alt="DHARMa plots for the 'Molecular Layer' model"></a></p>
</figure>
</div>
</div>
<div id="tabset-2-3" class="tab-pane" role="tabpanel" aria-labelledby="tabset-2-3-tab">
<div class="quarto-figure quarto-figure-center">
<figure class="figure"><p><a href="mod_ml_age_ppc_stats.png" class="lightbox" data-gallery="quarto-lightbox-gallery-7"><img src="mod_ml_age_ppc_stats.png" class="img-fluid quarto-figure quarto-figure-center figure-img" alt="Posterior Predictive Checks on simple stats for the 'Molecular Layer' model"></a></p>
</figure>
</div>
</div>
</div>
</div>
</section>
</div>
</div>
</div>
</section><section id="hypotheses" class="level1"><h1>Hypotheses</h1>
<p>Contrasts for hypotheses of interest were tested with <a href="https://rvlenth.github.io/emmeans/">emmeans</a> <span class="citation" data-cites="emmeans">(<a href="#ref-emmeans" role="doc-biblioref">Lenth, 2022</a>)</span>, with the Holm method to adjust for multiple comparisons.</p>
<div class="tabset-margin-container"></div><div class="panel-tabset">
<ul class="nav nav-tabs" role="tablist">
<li class="nav-item" role="presentation"><a class="nav-link active" id="tabset-6-1-tab" data-bs-toggle="tab" data-bs-target="#tabset-6-1" role="tab" aria-controls="tabset-6-1" aria-selected="true">Soma</a></li>
<li class="nav-item" role="presentation"><a class="nav-link" id="tabset-6-2-tab" data-bs-toggle="tab" data-bs-target="#tabset-6-2" role="tab" aria-controls="tabset-6-2" aria-selected="false">Molecular Layer</a></li>
</ul>
<div class="tab-content">
<div id="tabset-6-1" class="tab-pane active" role="tabpanel" aria-labelledby="tabset-6-1-tab">
<p>Since Reelin neurons have high metabolism, we hypothesized a higher Bnip3 expression in the soma of LEC/MEC neurons in AD rats, and the difference should be higher in older rats. We used the ‘Soma’ model to test these hypotheses.</p>
<div class="tabset-margin-container"></div><div class="panel-tabset">
<ul class="nav nav-tabs" role="tablist">
<li class="nav-item" role="presentation"><a class="nav-link active" id="tabset-4-1-tab" data-bs-toggle="tab" data-bs-target="#tabset-4-1" role="tab" aria-controls="tabset-4-1" aria-selected="true">Genotype (AD vs control)</a></li>
<li class="nav-item" role="presentation"><a class="nav-link" id="tabset-4-2-tab" data-bs-toggle="tab" data-bs-target="#tabset-4-2" role="tab" aria-controls="tabset-4-2" aria-selected="false">Genotype | Age</a></li>
<li class="nav-item" role="presentation"><a class="nav-link" id="tabset-4-3-tab" data-bs-toggle="tab" data-bs-target="#tabset-4-3" role="tab" aria-controls="tabset-4-3" aria-selected="false">Genotype : Age</a></li>
</ul>
<div class="tab-content">
<div id="tabset-4-1" class="tab-pane active" role="tabpanel" aria-labelledby="tabset-4-1-tab">
<div class="cell" data-layout-align="center">
<div class="code-copy-outer-scaffold"><div class="sourceCode" id="cb3"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="fu">emmeans</span><span class="op">(</span><span class="va">mod_soma</span>, specs <span class="op">=</span> <span class="st">"genotype"</span>, type <span class="op">=</span> <span class="st">"response"</span><span class="op">)</span> <span class="op">|&gt;</span> </span>
<span>    <span class="fu">contrast</span><span class="op">(</span>method <span class="op">=</span> <span class="st">"pairwise"</span>, adjust <span class="op">=</span> <span class="st">"holm"</span>, infer <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span></span></code></pre></div><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></div>
</div>
<div class="code-copy-outer-scaffold"><div class="sourceCode" id="cb4"><pre class="sourceCode default code-with-copy"><code class="sourceCode default"><span id="cb4-1"><a href="#cb4-1" aria-hidden="true" tabindex="-1"></a> contrast    ratio    SE  df asymp.LCL asymp.UCL null z.ratio p.value</span>
<span id="cb4-2"><a href="#cb4-2" aria-hidden="true" tabindex="-1"></a> WT / McGill 0.943 0.202 Inf     0.619      1.44    1  -0.274  0.7842</span>
<span id="cb4-3"><a href="#cb4-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb4-4"><a href="#cb4-4" aria-hidden="true" tabindex="-1"></a>Results are averaged over the levels of: age, area </span>
<span id="cb4-5"><a href="#cb4-5" aria-hidden="true" tabindex="-1"></a>Confidence level used: 0.95 </span>
<span id="cb4-6"><a href="#cb4-6" aria-hidden="true" tabindex="-1"></a>Intervals are back-transformed from the log scale </span>
<span id="cb4-7"><a href="#cb4-7" aria-hidden="true" tabindex="-1"></a>Tests are performed on the log scale </span></code></pre></div><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></div>
<div class="quarto-figure quarto-figure-center">
<figure class="figure"><p><a href="mod_soma_genotype.png" class="lightbox" data-gallery="quarto-lightbox-gallery-8"><img src="mod_soma_genotype.png" class="quarto-figure quarto-figure-center figure-img" alt="Annotated boxplots comparing BNIP3 expression in AD and control rats across age groups" height="500"></a></p>
</figure>
</div>
</div>
<div id="tabset-4-2" class="tab-pane" role="tabpanel" aria-labelledby="tabset-4-2-tab">
<div class="cell" data-layout-align="center">
<div class="code-copy-outer-scaffold"><div class="sourceCode" id="cb5"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="fu">emmeans</span><span class="op">(</span><span class="va">mod_soma</span>, specs <span class="op">=</span> <span class="op">~</span> <span class="va">genotype</span> <span class="op">|</span> <span class="va">age</span>, type <span class="op">=</span> <span class="st">"response"</span><span class="op">)</span> <span class="op">|&gt;</span> </span>
<span>    <span class="fu">contrast</span><span class="op">(</span>method <span class="op">=</span> <span class="st">"pairwise"</span>, adjust <span class="op">=</span> <span class="st">"none"</span>, infer <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span> <span class="op">|&gt;</span> </span>
<span>    <span class="fu"><a href="https://rdrr.io/r/base/as.data.frame.html">as.data.frame</a></span><span class="op">(</span><span class="op">)</span> <span class="op">|&gt;</span> </span>
<span>    <span class="fu">modify_at</span><span class="op">(</span><span class="st">"p.value"</span>, \<span class="op">(</span><span class="va">x</span><span class="op">)</span> <span class="fu"><a href="https://rdrr.io/r/stats/p.adjust.html">p.adjust</a></span><span class="op">(</span><span class="va">x</span>, method <span class="op">=</span> <span class="st">"holm"</span><span class="op">)</span><span class="op">)</span></span></code></pre></div><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></div>
</div>
<div class="code-copy-outer-scaffold"><div class="sourceCode" id="cb6"><pre class="sourceCode default code-with-copy"><code class="sourceCode default"><span id="cb6-1"><a href="#cb6-1" aria-hidden="true" tabindex="-1"></a>age = M3:</span>
<span id="cb6-2"><a href="#cb6-2" aria-hidden="true" tabindex="-1"></a> contrast     ratio     SE  df asymp.LCL asymp.UCL null z.ratio p.value</span>
<span id="cb6-3"><a href="#cb6-3" aria-hidden="true" tabindex="-1"></a> WT / McGill 0.5012 0.1862 Inf    0.2420     1.038    1  -1.859  0.1891</span>
<span id="cb6-4"><a href="#cb6-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb6-5"><a href="#cb6-5" aria-hidden="true" tabindex="-1"></a>age = M12:</span>
<span id="cb6-6"><a href="#cb6-6" aria-hidden="true" tabindex="-1"></a> contrast     ratio     SE  df asymp.LCL asymp.UCL null z.ratio p.value</span>
<span id="cb6-7"><a href="#cb6-7" aria-hidden="true" tabindex="-1"></a> WT / McGill 1.5831 0.5882 Inf    0.7643     3.279    1   1.236  0.4326</span>
<span id="cb6-8"><a href="#cb6-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb6-9"><a href="#cb6-9" aria-hidden="true" tabindex="-1"></a>age = M18:</span>
<span id="cb6-10"><a href="#cb6-10" aria-hidden="true" tabindex="-1"></a> contrast     ratio     SE  df asymp.LCL asymp.UCL null z.ratio p.value</span>
<span id="cb6-11"><a href="#cb6-11" aria-hidden="true" tabindex="-1"></a> WT / McGill 1.0566 0.3927 Inf    0.5100     2.189    1   0.148  0.8822</span>
<span id="cb6-12"><a href="#cb6-12" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb6-13"><a href="#cb6-13" aria-hidden="true" tabindex="-1"></a>Results are averaged over the levels of: area </span>
<span id="cb6-14"><a href="#cb6-14" aria-hidden="true" tabindex="-1"></a>Confidence level used: 0.95 </span>
<span id="cb6-15"><a href="#cb6-15" aria-hidden="true" tabindex="-1"></a>Intervals are back-transformed from the log scale </span>
<span id="cb6-16"><a href="#cb6-16" aria-hidden="true" tabindex="-1"></a>Tests are performed on the log scale </span></code></pre></div><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></div>
<div class="quarto-figure quarto-figure-center">
<figure class="figure"><p><a href="mod_soma_genotype_age.png" class="lightbox" data-gallery="quarto-lightbox-gallery-9"><img src="mod_soma_genotype_age.png" class="img-fluid quarto-figure quarto-figure-center figure-img" alt="Annotated boxplots comparing BNIP3 expression in AD and control rats within age groups"></a></p>
</figure>
</div>
</div>
<div id="tabset-4-3" class="tab-pane" role="tabpanel" aria-labelledby="tabset-4-3-tab">
<div class="cell" data-layout-align="center">
<div class="code-copy-outer-scaffold"><div class="sourceCode" id="cb7"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="fu">emmeans</span><span class="op">(</span><span class="va">mod_soma</span>, specs <span class="op">=</span> <span class="op">~</span> <span class="va">genotype</span> <span class="op">|</span> <span class="va">age</span>, type <span class="op">=</span> <span class="st">"response"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu">contrast</span><span class="op">(</span>interaction <span class="op">=</span> <span class="st">"pairwise"</span>, by <span class="op">=</span> <span class="cn">NULL</span>, adjust <span class="op">=</span> <span class="st">"none"</span>, infer <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span> <span class="op">|&gt;</span> </span>
<span>    <span class="fu"><a href="https://rdrr.io/r/base/as.data.frame.html">as.data.frame</a></span><span class="op">(</span><span class="op">)</span> <span class="op">|&gt;</span> </span>
<span>    <span class="fu">modify_at</span><span class="op">(</span><span class="st">"p.value"</span>, \<span class="op">(</span><span class="va">x</span><span class="op">)</span> <span class="fu"><a href="https://rdrr.io/r/stats/p.adjust.html">p.adjust</a></span><span class="op">(</span><span class="va">x</span>, method <span class="op">=</span> <span class="st">"holm"</span><span class="op">)</span><span class="op">)</span></span></code></pre></div><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></div>
</div>
<div class="code-copy-outer-scaffold"><div class="sourceCode" id="cb8"><pre class="sourceCode default code-with-copy"><code class="sourceCode default"><span id="cb8-1"><a href="#cb8-1" aria-hidden="true" tabindex="-1"></a> genotype_pairwise age_pairwise  ratio     SE  df asymp.LCL asymp.UCL null z.ratio p.value</span>
<span id="cb8-2"><a href="#cb8-2" aria-hidden="true" tabindex="-1"></a> WT / McGill        M3 / M12     0.3166 0.1664 Inf    0.1131     0.887    1  -2.189  0.0858</span>
<span id="cb8-3"><a href="#cb8-3" aria-hidden="true" tabindex="-1"></a> WT / McGill        M3 / M18     0.4744 0.2493 Inf    0.1694     1.329    1  -1.419  0.3118</span>
<span id="cb8-4"><a href="#cb8-4" aria-hidden="true" tabindex="-1"></a> WT / McGill        M12 / M18    1.4983 0.7874 Inf    0.5349     4.197    1   0.769  0.4417</span>
<span id="cb8-5"><a href="#cb8-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb8-6"><a href="#cb8-6" aria-hidden="true" tabindex="-1"></a>Results are averaged over the levels of: area </span>
<span id="cb8-7"><a href="#cb8-7" aria-hidden="true" tabindex="-1"></a>Confidence level used: 0.95 </span>
<span id="cb8-8"><a href="#cb8-8" aria-hidden="true" tabindex="-1"></a>Intervals are back-transformed from the log scale </span>
<span id="cb8-9"><a href="#cb8-9" aria-hidden="true" tabindex="-1"></a>Tests are performed on the log scale </span></code></pre></div><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></div>
<div class="quarto-figure quarto-figure-center">
<figure class="figure"><p><a href="mod_soma_genotype_age_inter.png" class="lightbox" data-gallery="quarto-lightbox-gallery-10"><img src="mod_soma_genotype_age_inter.png" class="img-fluid quarto-figure quarto-figure-center figure-img" alt="Annotated boxplots comparing BNIP3 expression in AD and control rats between age groups"></a></p>
</figure>
</div>
</div>
</div>
</div>
</div>
<div id="tabset-6-2" class="tab-pane" role="tabpanel" aria-labelledby="tabset-6-2-tab">
<p>We hypothesized that BNIP3 expression would be relatively low in other areas, but may be high in the projection areas of those neurons (CA3 &amp; dentate gyrus). We used the ‘Molecular Layer’ model to test these hypotheses.</p>
<div class="tabset-margin-container"></div><div class="panel-tabset">
<ul class="nav nav-tabs" role="tablist">
<li class="nav-item" role="presentation"><a class="nav-link active" id="tabset-5-1-tab" data-bs-toggle="tab" data-bs-target="#tabset-5-1" role="tab" aria-controls="tabset-5-1" aria-selected="true">Genotype (AD vs control)</a></li>
<li class="nav-item" role="presentation"><a class="nav-link" id="tabset-5-2-tab" data-bs-toggle="tab" data-bs-target="#tabset-5-2" role="tab" aria-controls="tabset-5-2" aria-selected="false">Genotype | Age</a></li>
<li class="nav-item" role="presentation"><a class="nav-link" id="tabset-5-3-tab" data-bs-toggle="tab" data-bs-target="#tabset-5-3" role="tab" aria-controls="tabset-5-3" aria-selected="false">Genotype : Age</a></li>
</ul>
<div class="tab-content">
<div id="tabset-5-1" class="tab-pane active" role="tabpanel" aria-labelledby="tabset-5-1-tab">
<div class="cell" data-layout-align="center">
<div class="code-copy-outer-scaffold"><div class="sourceCode" id="cb9"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="fu">emmeans</span><span class="op">(</span><span class="va">mod_ml</span>, specs <span class="op">=</span> <span class="st">"genotype"</span>, type <span class="op">=</span> <span class="st">"response"</span><span class="op">)</span> <span class="op">|&gt;</span> </span>
<span>    <span class="fu">contrast</span><span class="op">(</span>method <span class="op">=</span> <span class="st">"pairwise"</span>, adjust <span class="op">=</span> <span class="st">"holm"</span>, infer <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span></span></code></pre></div><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></div>
</div>
<div class="code-copy-outer-scaffold"><div class="sourceCode" id="cb10"><pre class="sourceCode default code-with-copy"><code class="sourceCode default"><span id="cb10-1"><a href="#cb10-1" aria-hidden="true" tabindex="-1"></a> contrast    ratio     SE  df asymp.LCL asymp.UCL null z.ratio p.value</span>
<span id="cb10-2"><a href="#cb10-2" aria-hidden="true" tabindex="-1"></a> WT / McGill 0.715 0.0999 Inf     0.543      0.94    1  -2.403  0.0163</span>
<span id="cb10-3"><a href="#cb10-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb10-4"><a href="#cb10-4" aria-hidden="true" tabindex="-1"></a>Results are averaged over the levels of: age </span>
<span id="cb10-5"><a href="#cb10-5" aria-hidden="true" tabindex="-1"></a>Confidence level used: 0.95 </span>
<span id="cb10-6"><a href="#cb10-6" aria-hidden="true" tabindex="-1"></a>Intervals are back-transformed from the log scale </span>
<span id="cb10-7"><a href="#cb10-7" aria-hidden="true" tabindex="-1"></a>Tests are performed on the log scale</span></code></pre></div><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></div>
<div class="quarto-figure quarto-figure-center">
<figure class="figure"><p><a href="mod_ml_genotype.png" class="lightbox" data-gallery="quarto-lightbox-gallery-11"><img src="mod_ml_genotype.png" class="quarto-figure quarto-figure-center figure-img" alt="Annotated boxplots comparing Optical Density in AD vs control rats for all age groups" height="500"></a></p>
</figure>
</div>
</div>
<div id="tabset-5-2" class="tab-pane" role="tabpanel" aria-labelledby="tabset-5-2-tab">
<div class="cell" data-layout-align="center">
<div class="code-copy-outer-scaffold"><div class="sourceCode" id="cb11"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="fu">emmeans</span><span class="op">(</span><span class="va">mod_ml</span>, specs <span class="op">=</span> <span class="op">~</span> <span class="va">genotype</span> <span class="op">|</span> <span class="va">age</span>, type <span class="op">=</span> <span class="st">"response"</span><span class="op">)</span> <span class="op">|&gt;</span> </span>
<span>    <span class="fu">contrast</span><span class="op">(</span>method <span class="op">=</span> <span class="st">"pairwise"</span>, adjust <span class="op">=</span> <span class="st">"none"</span>, infer <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span> <span class="op">|&gt;</span> </span>
<span>    <span class="fu"><a href="https://rdrr.io/r/base/as.data.frame.html">as.data.frame</a></span><span class="op">(</span><span class="op">)</span> <span class="op">|&gt;</span> </span>
<span>    <span class="fu">modify_at</span><span class="op">(</span><span class="st">"p.value"</span>, \<span class="op">(</span><span class="va">x</span><span class="op">)</span> <span class="fu"><a href="https://rdrr.io/r/stats/p.adjust.html">p.adjust</a></span><span class="op">(</span><span class="va">x</span>, method <span class="op">=</span> <span class="st">"holm"</span><span class="op">)</span><span class="op">)</span></span></code></pre></div><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></div>
</div>
<div class="code-copy-outer-scaffold"><div class="sourceCode" id="cb12"><pre class="sourceCode default code-with-copy"><code class="sourceCode default"><span id="cb12-1"><a href="#cb12-1" aria-hidden="true" tabindex="-1"></a>age = M3:</span>
<span id="cb12-2"><a href="#cb12-2" aria-hidden="true" tabindex="-1"></a> contrast     ratio     SE  df asymp.LCL asymp.UCL null z.ratio p.value</span>
<span id="cb12-3"><a href="#cb12-3" aria-hidden="true" tabindex="-1"></a> WT / McGill 0.5610 0.1535 Inf    0.3281    0.9592    1  -2.112  0.0694</span>
<span id="cb12-4"><a href="#cb12-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb12-5"><a href="#cb12-5" aria-hidden="true" tabindex="-1"></a>age = M12:</span>
<span id="cb12-6"><a href="#cb12-6" aria-hidden="true" tabindex="-1"></a> contrast     ratio     SE  df asymp.LCL asymp.UCL null z.ratio p.value</span>
<span id="cb12-7"><a href="#cb12-7" aria-hidden="true" tabindex="-1"></a> WT / McGill 1.2950 0.3529 Inf    0.7591    2.2091    1   0.949  0.3429</span>
<span id="cb12-8"><a href="#cb12-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb12-9"><a href="#cb12-9" aria-hidden="true" tabindex="-1"></a>age = M18:</span>
<span id="cb12-10"><a href="#cb12-10" aria-hidden="true" tabindex="-1"></a> contrast     ratio     SE  df asymp.LCL asymp.UCL null z.ratio p.value</span>
<span id="cb12-11"><a href="#cb12-11" aria-hidden="true" tabindex="-1"></a> WT / McGill 0.5021 0.0823 Inf    0.3641    0.6924    1  -4.202  0.0001</span>
<span id="cb12-12"><a href="#cb12-12" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb12-13"><a href="#cb12-13" aria-hidden="true" tabindex="-1"></a>Confidence level used: 0.95 </span>
<span id="cb12-14"><a href="#cb12-14" aria-hidden="true" tabindex="-1"></a>Intervals are back-transformed from the log scale </span>
<span id="cb12-15"><a href="#cb12-15" aria-hidden="true" tabindex="-1"></a>Tests are performed on the log scale </span></code></pre></div><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></div>
<p><a href="mod_ml_genotype_age.png" class="lightbox" data-gallery="quarto-lightbox-gallery-12"><img src="mod_ml_genotype_age.png" class="img-fluid" alt="Annotated boxplots comparing Optical Density in AD vs control rats within age groups"></a></p>
</div>
<div id="tabset-5-3" class="tab-pane" role="tabpanel" aria-labelledby="tabset-5-3-tab">
<div class="cell" data-layout-align="center">
<div class="code-copy-outer-scaffold"><div class="sourceCode" id="cb13"><pre class="downlit sourceCode r code-with-copy"><code class="sourceCode R"><span><span class="fu">emmeans</span><span class="op">(</span><span class="va">mod_ml</span>, specs <span class="op">=</span> <span class="op">~</span> <span class="va">genotype</span> <span class="op">|</span> <span class="va">age</span>, type <span class="op">=</span> <span class="st">"response"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu">contrast</span><span class="op">(</span>interaction <span class="op">=</span> <span class="st">"pairwise"</span>, by <span class="op">=</span> <span class="cn">NULL</span>, adjust <span class="op">=</span> <span class="st">"none"</span>, infer <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span> <span class="op">|&gt;</span> </span>
<span>    <span class="fu"><a href="https://rdrr.io/r/base/as.data.frame.html">as.data.frame</a></span><span class="op">(</span><span class="op">)</span> <span class="op">|&gt;</span> </span>
<span>    <span class="fu">modify_at</span><span class="op">(</span><span class="st">"p.value"</span>, \<span class="op">(</span><span class="va">x</span><span class="op">)</span> <span class="fu"><a href="https://rdrr.io/r/stats/p.adjust.html">p.adjust</a></span><span class="op">(</span><span class="va">x</span>, method <span class="op">=</span> <span class="st">"holm"</span><span class="op">)</span><span class="op">)</span></span></code></pre></div><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></div>
</div>
<div class="code-copy-outer-scaffold"><div class="sourceCode" id="cb14"><pre class="sourceCode default code-with-copy"><code class="sourceCode default"><span id="cb14-1"><a href="#cb14-1" aria-hidden="true" tabindex="-1"></a> genotype_pairwise age_pairwise  ratio     SE  df asymp.LCL asymp.UCL null z.ratio p.value</span>
<span id="cb14-2"><a href="#cb14-2" aria-hidden="true" tabindex="-1"></a> WT / McGill        M3 / M12     0.4332 0.1673 Inf    0.2032     0.924    1  -2.166  0.0606</span>
<span id="cb14-3"><a href="#cb14-3" aria-hidden="true" tabindex="-1"></a> WT / McGill        M3 / M18     1.1172 0.3564 Inf    0.5978     2.088    1   0.347  0.7283</span>
<span id="cb14-4"><a href="#cb14-4" aria-hidden="true" tabindex="-1"></a> WT / McGill        M12 / M18    2.5788 0.8202 Inf    1.3826     4.810    1   2.979  0.0087</span>
<span id="cb14-5"><a href="#cb14-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb14-6"><a href="#cb14-6" aria-hidden="true" tabindex="-1"></a>Confidence level used: 0.95 </span>
<span id="cb14-7"><a href="#cb14-7" aria-hidden="true" tabindex="-1"></a>Intervals are back-transformed from the log scale </span>
<span id="cb14-8"><a href="#cb14-8" aria-hidden="true" tabindex="-1"></a>Tests are performed on the log scale </span></code></pre></div><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></div>
<p><a href="mod_ml_genotype_age_inter.png" class="lightbox" data-gallery="quarto-lightbox-gallery-13"><img src="mod_ml_genotype_age_inter.png" class="img-fluid" alt="Annotated boxplots comparing Optical Density in AD vs control rats between age groups"></a></p>
</div>
</div>
</div>
</div>
</div>
</div>




</section><a onclick="window.scrollTo(0, 0); return false;" role="button" id="quarto-back-to-top"><i class="bi bi-arrow-up"></i> Back to top</a><div id="quarto-appendix" class="default"><section class="quarto-appendix-contents" role="doc-bibliography" id="quarto-bibliography"><h2 class="anchored quarto-appendix-heading">References</h2><div id="refs" class="references csl-bib-body hanging-indent" data-entry-spacing="0" data-line-spacing="2" role="list">
<div id="ref-glmmTMB" class="csl-entry" role="listitem">
Brooks, M. E., Kristensen, K., van Benthem, K. J., Magnusson, A., Berg, C. W., Nielsen, A., Skaug, H. J., Maechler, M., &amp; Bolker, B. M. (2017). <span class="nocase">glmmTMB</span> balances speed and flexibility among packages for zero-inflated generalized linear mixed modeling. <em>The R Journal</em>, <em>9</em>(2), 378–400. <a href="https://journal.r-project.org/archive/2017/RJ-2017-066/index.html">https://journal.r-project.org/archive/2017/RJ-2017-066/index.html</a>
</div>
<div id="ref-DHARMa" class="csl-entry" role="listitem">
Hartig, F. (2022). <em>DHARMa: Residual diagnostics for hierarchical (multi-level / mixed) regression models</em>. <a href="https://CRAN.R-project.org/package=DHARMa">https://CRAN.R-project.org/package=DHARMa</a>
</div>
<div id="ref-emmeans" class="csl-entry" role="listitem">
Lenth, R. V. (2022). <em>Emmeans: Estimated marginal means, aka least-squares means</em>. <a href="https://CRAN.R-project.org/package=emmeans">https://CRAN.R-project.org/package=emmeans</a>
</div>
<div id="ref-performance" class="csl-entry" role="listitem">
Lüdecke, D., Ben-Shachar, M. S., Patil, I., Waggoner, P., &amp; Makowski, D. (2021). <span class="nocase">performance</span>: An <span>R</span> package for assessment, comparison and testing of statistical models. <em>Journal of Open Source Software</em>, <em>6</em>(60), 3139. <a href="https://doi.org/10.21105/joss.03139">https://doi.org/10.21105/joss.03139</a>
</div>
<div id="ref-rodriguez-dubocBnip3ExpressionBrain2025" class="csl-entry" role="listitem">
Rodriguez-Duboc, A., Velicer, S. T., &amp; Kobro-Flatmoen, A. (2025). Bnip3 expression in the brain of an <span>Alzheimer</span>’s disease rat model. <em>Brain Mechanisms</em>, <em>148–150</em>, 202518. <a href="https://doi.org/10.1016/j.bramec.2025.202518">https://doi.org/10.1016/j.bramec.2025.202518</a>
</div>
</div></section></div></main><!-- /main --><script id="quarto-html-after-body" type="application/javascript">
  window.document.addEventListener("DOMContentLoaded", function (event) {
    // Ensure there is a toggle, if there isn't float one in the top right
    if (window.document.querySelector('.quarto-color-scheme-toggle') === null) {
      const a = window.document.createElement('a');
      a.classList.add('top-right');
      a.classList.add('quarto-color-scheme-toggle');
      a.href = "";
      a.onclick = function() { try { window.quartoToggleColorScheme(); } catch {} return false; };
      const i = window.document.createElement("i");
      i.classList.add('bi');
      a.appendChild(i);
      window.document.body.appendChild(a);
    }
    setColorSchemeToggle(hasAlternateSentinel())
    const isCodeAnnotation = (el) => {
      for (const clz of el.classList) {
        if (clz.startsWith('code-annotation-')) {                     
          return true;
        }
      }
      return false;
    }
    const onCopySuccess = function(e) {
      // button target
      const button = e.trigger;
      // don't keep focus
      button.blur();
      // flash "checked"
      button.classList.add('code-copy-button-checked');
      var currentTitle = button.getAttribute("title");
      button.setAttribute("title", "Copied!");
      let tooltip;
      if (window.bootstrap) {
        button.setAttribute("data-bs-toggle", "tooltip");
        button.setAttribute("data-bs-placement", "left");
        button.setAttribute("data-bs-title", "Copied!");
        tooltip = new bootstrap.Tooltip(button, 
          { trigger: "manual", 
            customClass: "code-copy-button-tooltip",
            offset: [0, -8]});
        tooltip.show();    
      }
      setTimeout(function() {
        if (tooltip) {
          tooltip.hide();
          button.removeAttribute("data-bs-title");
          button.removeAttribute("data-bs-toggle");
          button.removeAttribute("data-bs-placement");
        }
        button.setAttribute("title", currentTitle);
        button.classList.remove('code-copy-button-checked');
      }, 1000);
      // clear code selection
      e.clearSelection();
    }
    const getTextToCopy = function(trigger) {
        const codeEl = trigger.previousElementSibling.cloneNode(true);
        for (const childEl of codeEl.children) {
          if (isCodeAnnotation(childEl)) {
            childEl.remove();
          }
        }
        return codeEl.innerText;
    }
    const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
      text: getTextToCopy
    });
    clipboard.on('success', onCopySuccess);
    if (window.document.getElementById('quarto-embedded-source-code-modal')) {
      const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
        text: getTextToCopy,
        container: window.document.getElementById('quarto-embedded-source-code-modal')
      });
      clipboardModal.on('success', onCopySuccess);
    }
      var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
      var mailtoRegex = new RegExp(/^mailto:/);
        var filterRegex = new RegExp("https:\/\/ma-riviere\.com");
      var isInternal = (href) => {
          return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
      }
      // Inspect non-navigation links and adorn them if external
     var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
      for (var i=0; i<links.length; i++) {
        const link = links[i];
        if (!isInternal(link.href)) {
          // undo the damage that might have been done by quarto-nav.js in the case of
          // links that we want to consider external
          if (link.dataset.originalHref !== undefined) {
            link.href = link.dataset.originalHref;
          }
            // target, if specified
            link.setAttribute("target", "_blank");
            if (link.getAttribute("rel") === null) {
              link.setAttribute("rel", "noopener");
            }
            // default icon
            link.classList.add("external");
        }
      }
    function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
      const config = {
        allowHTML: true,
        maxWidth: 500,
        delay: 100,
        arrow: false,
        appendTo: function(el) {
            return el.parentElement;
        },
        interactive: true,
        interactiveBorder: 10,
        theme: 'quarto',
        placement: 'bottom-start',
      };
      if (contentFn) {
        config.content = contentFn;
      }
      if (onTriggerFn) {
        config.onTrigger = onTriggerFn;
      }
      if (onUntriggerFn) {
        config.onUntrigger = onUntriggerFn;
      }
      window.tippy(el, config); 
    }
    const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
    for (var i=0; i<noterefs.length; i++) {
      const ref = noterefs[i];
      tippyHover(ref, function() {
        // use id or data attribute instead here
        let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
        try { href = new URL(href).hash; } catch {}
        const id = href.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note) {
          return note.innerHTML;
        } else {
          return "";
        }
      });
    }
    const xrefs = window.document.querySelectorAll('a.quarto-xref');
    const processXRef = (id, note) => {
      // Strip column container classes
      const stripColumnClz = (el) => {
        el.classList.remove("page-full", "page-columns");
        if (el.children) {
          for (const child of el.children) {
            stripColumnClz(child);
          }
        }
      }
      stripColumnClz(note)
      if (id === null || id.startsWith('sec-')) {
        // Special case sections, only their first couple elements
        const container = document.createElement("div");
        if (note.children && note.children.length > 2) {
          container.appendChild(note.children[0].cloneNode(true));
          for (let i = 1; i < note.children.length; i++) {
            const child = note.children[i];
            if (child.tagName === "P" && child.innerText === "") {
              continue;
            } else {
              container.appendChild(child.cloneNode(true));
              break;
            }
          }
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(container);
          }
          return container.innerHTML
        } else {
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(note);
          }
          return note.innerHTML;
        }
      } else {
        // Remove any anchor links if they are present
        const anchorLink = note.querySelector('a.anchorjs-link');
        if (anchorLink) {
          anchorLink.remove();
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        if (note.classList.contains("callout")) {
          return note.outerHTML;
        } else {
          return note.innerHTML;
        }
      }
    }
    for (var i=0; i<xrefs.length; i++) {
      const xref = xrefs[i];
      tippyHover(xref, undefined, function(instance) {
        instance.disable();
        let url = xref.getAttribute('href');
        let hash = undefined; 
        if (url.startsWith('#')) {
          hash = url;
        } else {
          try { hash = new URL(url).hash; } catch {}
        }
        if (hash) {
          const id = hash.replace(/^#\/?/, "");
          const note = window.document.getElementById(id);
          if (note !== null) {
            try {
              const html = processXRef(id, note.cloneNode(true));
              instance.setContent(html);
            } finally {
              instance.enable();
              instance.show();
            }
          } else {
            // See if we can fetch this
            fetch(url.split('#')[0])
            .then(res => res.text())
            .then(html => {
              const parser = new DOMParser();
              const htmlDoc = parser.parseFromString(html, "text/html");
              const note = htmlDoc.getElementById(id);
              if (note !== null) {
                const html = processXRef(id, note);
                instance.setContent(html);
              } 
            }).finally(() => {
              instance.enable();
              instance.show();
            });
          }
        } else {
          // See if we can fetch a full url (with no hash to target)
          // This is a special case and we should probably do some content thinning / targeting
          fetch(url)
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.querySelector('main.content');
            if (note !== null) {
              // This should only happen for chapter cross references
              // (since there is no id in the URL)
              // remove the first header
              if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
                note.children[0].remove();
              }
              const html = processXRef(null, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      }, function(instance) {
      });
    }
        let selectedAnnoteEl;
        const selectorForAnnotation = ( cell, annotation) => {
          let cellAttr = 'data-code-cell="' + cell + '"';
          let lineAttr = 'data-code-annotation="' +  annotation + '"';
          const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
          return selector;
        }
        const selectCodeLines = (annoteEl) => {
          const doc = window.document;
          const targetCell = annoteEl.getAttribute("data-target-cell");
          const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
          const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
          const lines = annoteSpan.getAttribute("data-code-lines").split(",");
          const lineIds = lines.map((line) => {
            return targetCell + "-" + line;
          })
          let top = null;
          let height = null;
          let parent = null;
          if (lineIds.length > 0) {
              //compute the position of the single el (top and bottom and make a div)
              const el = window.document.getElementById(lineIds[0]);
              top = el.offsetTop;
              height = el.offsetHeight;
              parent = el.parentElement.parentElement;
            if (lineIds.length > 1) {
              const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
              const bottom = lastEl.offsetTop + lastEl.offsetHeight;
              height = bottom - top;
            }
            if (top !== null && height !== null && parent !== null) {
              // cook up a div (if necessary) and position it 
              let div = window.document.getElementById("code-annotation-line-highlight");
              if (div === null) {
                div = window.document.createElement("div");
                div.setAttribute("id", "code-annotation-line-highlight");
                div.style.position = 'absolute';
                parent.appendChild(div);
              }
              div.style.top = top - 2 + "px";
              div.style.height = height + 4 + "px";
              div.style.left = 0;
              let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
              if (gutterDiv === null) {
                gutterDiv = window.document.createElement("div");
                gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
                gutterDiv.style.position = 'absolute';
                const codeCell = window.document.getElementById(targetCell);
                const gutter = codeCell.querySelector('.code-annotation-gutter');
                gutter.appendChild(gutterDiv);
              }
              gutterDiv.style.top = top - 2 + "px";
              gutterDiv.style.height = height + 4 + "px";
            }
            selectedAnnoteEl = annoteEl;
          }
        };
        const unselectCodeLines = () => {
          const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
          elementsIds.forEach((elId) => {
            const div = window.document.getElementById(elId);
            if (div) {
              div.remove();
            }
          });
          selectedAnnoteEl = undefined;
        };
          // Handle positioning of the toggle
      window.addEventListener(
        "resize",
        throttle(() => {
          elRect = undefined;
          if (selectedAnnoteEl) {
            selectCodeLines(selectedAnnoteEl);
          }
        }, 10)
      );
      function throttle(fn, ms) {
      let throttle = false;
      let timer;
        return (...args) => {
          if(!throttle) { // first call gets through
              fn.apply(this, args);
              throttle = true;
          } else { // all the others get throttled
              if(timer) clearTimeout(timer); // cancel #2
              timer = setTimeout(() => {
                fn.apply(this, args);
                timer = throttle = false;
              }, ms);
          }
        };
      }
        // Attach click handler to the DT
        const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
        for (const annoteDlNode of annoteDls) {
          annoteDlNode.addEventListener('click', (event) => {
            const clickedEl = event.target;
            if (clickedEl !== selectedAnnoteEl) {
              unselectCodeLines();
              const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
              if (activeEl) {
                activeEl.classList.remove('code-annotation-active');
              }
              selectCodeLines(clickedEl);
              clickedEl.classList.add('code-annotation-active');
            } else {
              // Unselect the line
              unselectCodeLines();
              clickedEl.classList.remove('code-annotation-active');
            }
          });
        }
    const findCites = (el) => {
      const parentEl = el.parentElement;
      if (parentEl) {
        const cites = parentEl.dataset.cites;
        if (cites) {
          return {
            el,
            cites: cites.split(' ')
          };
        } else {
          return findCites(el.parentElement)
        }
      } else {
        return undefined;
      }
    };
    var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
    for (var i=0; i<bibliorefs.length; i++) {
      const ref = bibliorefs[i];
      const citeInfo = findCites(ref);
      if (citeInfo) {
        tippyHover(citeInfo.el, function() {
          var popup = window.document.createElement('div');
          citeInfo.cites.forEach(function(cite) {
            var citeDiv = window.document.createElement('div');
            citeDiv.classList.add('hanging-indent');
            citeDiv.classList.add('csl-entry');
            var biblioDiv = window.document.getElementById('ref-' + cite);
            if (biblioDiv) {
              citeDiv.innerHTML = biblioDiv.innerHTML;
            }
            popup.appendChild(citeDiv);
          });
          return popup.innerHTML;
        });
      }
    }
  });
  </script>
</div> <!-- /content -->
<footer class="footer"><div class="nav-footer">
    <div class="nav-footer-left">
      <ul class="footer-items list-unstyled">
<li class="nav-item">
    <a class="nav-link" href="../../../LICENSE.md">
<p>© 2021-2025, Marc-Aurèle Rivière</p>
</a>
  </li>  
</ul>
</div>   
    <div class="nav-footer-center">

<div class="cookie-consent-footer"><a href="#" id="open_preferences_center">Cookie Preferences</a></div></div>
    <div class="nav-footer-right">
      <ul class="footer-items list-unstyled">
<li class="nav-item">
    <a class="nav-link" href="https://quarto.org/">
<p>Made with <i class="fa-brands fa-r-project" aria-label="r-project"></i>, <i class="fa-brands fa-js" aria-label="js"></i>, and <a href="https://quarto.org/">Quarto</a> ♥️</p>
</a>
  </li>  
</ul>
</div>
  </div>
</footer><script>var lightboxQuarto = GLightbox({"closeEffect":"zoom","descPosition":"bottom","loop":false,"openEffect":"zoom","selector":".lightbox"});
(function() {
  let previousOnload = window.onload;
  window.onload = () => {
    if (previousOnload) {
      previousOnload();
    }
    lightboxQuarto.on('slide_before_load', (data) => {
      const { slideIndex, slideNode, slideConfig, player, trigger } = data;
      const href = trigger.getAttribute('href');
      if (href !== null) {
        const imgEl = window.document.querySelector(`a[href="${href}"] img`);
        if (imgEl !== null) {
          const srcAttr = imgEl.getAttribute("src");
          if (srcAttr && srcAttr.startsWith("data:")) {
            slideConfig.href = srcAttr;
          }
        }
      } 
    });
  
    lightboxQuarto.on('slide_after_load', (data) => {
      const { slideIndex, slideNode, slideConfig, player, trigger } = data;
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(slideNode);
      }
    });
  
  };
  
})();
          </script>


</body></html>